### DISCUSSION TITLE
Citation Key: how is 'type' function used

### DISCUSSION THREAD

**Discussion by slhenty** [retorquere/zotero-better-bibtex#2409] [Q&A] [answered]
I'm not seeing an example of how the type function is used to '...skip to the next pattern.' Anyone have an example of such usage?

**Comment by retorquere** [comment, marked as answer]
You can do something like `type(book,bookSection) + veryshorttitle + year | auth + year` which will use `veryshorttitle + year` for books and bookSections, and `auth + year` for anything else.
